dashboard/db/migrate/20200504010945_add_kind_to_parent_levels_child_levels.rb

```diff
@@ -0,0 +1,5 @@
+class AddKindToParentLevelsChildLevels < ActiveRecord::Migration[5.0]
+  def change
+    add_column :parent_levels_child_levels, :kind, :string, null: false, default: 'sublevel'
```

Comment:
Is default only used because the existing table needs to be backfilled? Just checking for my understanding.

Reply:
default is so that if you do not specify a value, you'll get a sublevel, rather than a validation error, in the next PR https://github.com/code-dot-org/code-dot-org/pull/34650 . the unit tests there attempt to illustrate this

Reply:
I think it's slightly better to require a value, but not enough to ask you to change anything. Would be an interesting topic to discuss for next time this comes up, if you feel differently.